datalad/core/local/create.py

```diff
@@ -116,10 +116,10 @@ class Create(Interface):
              code_py="create(dataset='.', path='mysubdataset')",
              code_cmd="datalad create -d . mysubdataset"),
         dict(text="Create a dataset in an existing, non-empty directory",
-             code_py="create(force=True, path='.')",
+             code_py="create(force=True)",
              code_cmd="datalad create --force"),
         dict(text="Create a plain Git repository",
-             code_py="create(path='mydataset', no_annex=True)",
+             code_py="create(no_annex=True)",
```

Comment:
I believe I made a mistake in the cmd example and forgot to give a path. Running `create(no_annex=True)` or `datalad create --no-annex` in any directory that is not empty would give an error ("refusing to create dataset in non-empty directory"), I think. We should either put a path in both examples, or improve the description to say something like "Create  a plain Git repository in an existing, empty directory". Personally, I more often create new datasets than transforming existing directories into datasets, so I have slight preference for keeping a path, but thats not a strong opinion.

Reply:
Yeah, I was assuming "existing and empty", but indeed that's not stated.  I agree with your preference and will go that way.